remake.yml

```diff
@@ -18,7 +18,10 @@ targets:
   main:
     depends:
       - state_tasks
+      - 2_process/src/tally_site_obs.R
       - 3_visualize/out/site_map.png
+      - 3_visualize/src/plot_site_data.R
```

Comment:
I just noticed that this is not right. The `depends` elements should be targets that your code creates, not code that you need to use to create targets.